### PR TITLE
125 add command line options

### PR DESCRIPTION
**Pull request by rodolakis**

### CODE REVIEW

**Comment by prjemian** [comment]
```bash
(mdaviz) prjemian@arf:~/.../BCDA-APS/mdaviz$ mdaviz -h
usage: mdaviz [-h] [--log {critical,fatal,error,warn,warning,info,debug,notset}] [-v] directory

mdaviz: Python Qt5 application to visualize mda data.

positional arguments:
  directory             Directory loaded at start up. This argument is required.

options:
  -h, --help            show this help message and exit
  --log {critical,fatal,error,warn,warning,info,debug,notset}
                        Provide logging level. Example '--log debug'. Default level: 'warning'
  -v, --version         show program's version number and exit


ERROR: You must specified the directory:
	mdaviz /path/to/mda/data
```

An error message is unexpected when asking for help.  The `argparse` package can handle this situation without the extra exception handler.  It uses the `help` text.

File: mdaviz/app.py [hunk 1]
```diff
@@ -1,27 +1,106 @@
...
+
 def main():  # for future command-line options
-    gui()
+    from pathlib import Path
+
+    global logger
+
+    try:
+        options = command_line_interface()
+    except SystemExit as e:
```
Comment (prjemian) [inline mdaviz/app.py:71]: `except` is not necessary

File: mdaviz/app.py [hunk 2]
```diff
@@ -1,27 +1,106 @@
...
+    return parser.parse_args()
+
+
 def main():  # for future command-line options
-    gui()
+    from pathlib import Path
+
+    global logger
+
+    try:
```
Comment (prjemian) [inline mdaviz/app.py:69]: `try` is not necessary

**Comment by prjemian** [comment]
This is how it should work, asking for help:

```bash
(mdaviz) prjemian@arf:~/.../BCDA-APS/mdaviz$ mdaviz -h
usage: mdaviz [-h] [--log {critical,fatal,error,warn,warning,info,debug,notset}] [-v] directory

mdaviz: Python Qt5 application to visualize mda data.

positional arguments:
  directory             Directory loaded at start up. This argument is required.

options:
  -h, --help            show this help message and exit
  --log {critical,fatal,error,warn,warning,info,debug,notset}
                        Provide logging level. Example '--log debug'. Default level: 'warning'
  -v, --version         show program's version number and exit
```

And then, run without the required argument (fails):

```bash
(mdaviz) prjemian@arf:~/.../BCDA-APS/mdaviz$ mdaviz 
usage: mdaviz [-h] [--log {critical,fatal,error,warn,warning,info,debug,notset}] [-v] directory
mdaviz: error: the following arguments are required: directory
```

**Comment by rodolakis** [comment]
I just found it not super clear enough like that. I wanted to show something more explicite (the error does not stand out much)

**Comment by rodolakis** [comment]
I had uncommented python 3.8 because the test unit was failing, took me a little bit to debug. I put it back.

It is ready now.

**Review by prjemian — APPROVED** [review]
LGTM

**Pull request merged**